grades/api.py

```diff
@@ -230,3 +232,33 @@ def generate_program_certificate(user, program):
         user.username,
         program.title
     )
+
+
+def update_combined_final_grade(user, course):
```

Comment:
Suggestion: add `_or_create` to the function name so the side effects are clearer